app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java

```diff
@@ -438,9 +438,6 @@ public void onClick(View view) {
         } else if (view.getId() == playbackPitchButton.getId()) {
             openPlaybackParameterDialog();
 
-        } else if (view.getId() == metadata.getId()) {
-            scrollToSelected();
```

Comment:
Would be better to change this line to `onOpenDetail(info.getServiceId(), info.getUrl(), info.getName())` rather than using a `setOnClickListener`, just to keep consistency.